components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/contract-storage.adoc

```diff
@@ -35,9 +35,9 @@ use starknet::syscalls::storage_write_syscall;
 
 The most common way to interact with a contract's storage is through storage variables.
 
-The `@storage_var` decorator declares a variable that will be kept as part of the contract storage. The variable can consist of a single felt, or it can be a mapping from multiple arguments to a tuple of felts or structs.
+The `#[storage]` attribute above the `Storage` struct declares that the content of this struct will be part of the contract storage. The storage variables stored inside this struct can consist of a single felt, or it can be a mapping from multiple arguments to a tuple of felts or structs.
```

Comment:
```suggestion
The `#[storage]` attribute above the `Storage` struct declares that the content of this struct are part of the contract storage. The storage variables stored inside this struct can consist of a single felt, or it can be a mapping from multiple arguments to a tuple of felts or structs.
```